Druid/DRInterpreterCompiler.class.st

```diff
@@ -0,0 +1,310 @@
+"
+Implements a basic interpreter to compiler.
+
+- All the generated bytecodes or primitives are unimplemented.
+- No error handling.
+
+This enables to build a VM which includes the JIT, but does not know how to JIT any method. Then the primitives or bytecodes could be added incrementally, and provides a baseline to compare performance between implementations.
+
+
+
+"
+Class {
+	#name : #DRInterpreterCompiler,
```

Comment:
This is not a Compiler. This uses the Compiler(Compiler). 
It's a Compilation or CompilationBuilder.